### ISSUE TITLE
How to configure Sharp?

### ISSUE THREAD

**Issue by stulnikov-roud** [RafidMuhymin/astro-imagetools#103]
I want to optimize images. So I need to configure globally quality for each format separately and enable mozjpeg for JPG images.
Is it possible?

**Comment by AndreaPontrandolfo** [comment]
Yes, i need the same too, atleast so we can configure the "effort" parameter.
